app/assets/javascripts/signature.js

```diff
@@ -78,7 +78,7 @@
     this.pos = this._newEvent(event).penPosition()
     this.context.beginPath()
     this.context.moveTo(this.pos.x, this.pos.y)
-    this._savePenStroke(this._continueStroke)
```

Comment:
`_newStroke` was never used for any of the events. Replacing `_continueStroke` here with `_continueStroke` provides the correct sigpath.